android/app/src/main/java/app/organicmaps/editor/OsmLoginFragment.java

```diff
@@ -78,6 +85,56 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 
     ScrollView scrollView = view.findViewById(R.id.scrollView);
     ViewCompat.setOnApplyWindowInsetsListener(scrollView, new ScrollableContentInsetsListener(scrollView));
+    mViewModel.setListener(this);
+    validateInput();
+    showUpdatedUIState();
+  }
+
+  private void validateInput()
```

Comment:
What is the value in creating a method whose content is part of the initialization anyway?

Reply:
Did not want to bloat the method `onViewCreated()`, so created a separate method.